Source/WebKit/Platform/spi/ios/PhotosUISPI.h

```diff
@@ -52,6 +56,20 @@
 
 @end
 
+@interface PHPickerConfiguration : NSObject
+
+@property (nonatomic, setter=_setAllowsDownscaling:) BOOL _allowsDownscaling SPI_AVAILABLE(ios(16));
+
+@end
+
+@interface PHPickerViewController : UIViewController
+
+@end
+
+@interface PHPickerResult : NSObject
+
+@end
```

Comment:
`PHPickerViewController` and `PHPickerResult` are API, and we're not using any SPI from these classes, so they do not need to be in an SPI header.